CalculatorInput/CalculatorInputView.m

```diff
@@ -11,6 +11,7 @@ @interface CalculatorInputView ()
 @implementation CalculatorInputView
 
 - (id)initWithFrame:(CGRect)frame {
+    self = [super initWithFrame:frame];
     self = [[[NSBundle bundleForClass:[self class]] loadNibNamed:@"CalculatorInputView" owner:self options:nil] firstObject];
     if (self) {
```

Comment:
There was a warning arising here (and would even propagate up to Venmo) about not calling `super`. I'm not sure what the right way is to have `-initWithFrame:` initialize itself from a nib these days but this seems innocuous enough, right? I originally tried just invoking `-[super initWithFrame:]` but that resulted in a new warning about not assigning the result to self.

Unless I'm forgetting something crucial about Obj-C initialization, setting self twice in this way should be okay, right?